### PR TITLE
Jenkins/RFS bug fixes for working pipeline

### PR DESCRIPTION
**Pull request by lewijacn**
### Description
Required fixes to get RFS flow working as well as fix minor Jenkins issues

image

### Issues Resolved
[List any issues this PR will resolve]

Is this a backport? If so, please add backport PR # and/or commits #

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MetadataMigration/src/main/java/com/rfs/MetadataMigration.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public static class Args {
         public String s3Region;
 
         @ParametersDelegate
-        public ConnectionDetails.TargetArgs targetArgs;
+        public ConnectionDetails.TargetArgs targetArgs = new ConnectionDetails.TargetArgs();
```
Comment (AndreKurait) [inline MetadataMigration/src/main/java/com/rfs/MetadataMigration.java:63]: @chelma, is this part of the change you're doing?

Comment (chelma) [reply MetadataMigration/src/main/java/com/rfs/MetadataMigration.java:63]: Yeah, I just pushed out a PR that does the same thing.

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/backfill_tests.py [hunk 2]
```diff
@@ -56,10 +56,14 @@ def setup_backfill(request):
...
-    backfill.start()
-    backfill.scale(units=10)
+    snapshot_result: CommandResult = snapshot.create(wait=True)
+    assert snapshot_result.success
+    metadata_result: CommandResult = metadata.migrate()
+    assert metadata_result.success
+    backfill_start_result: CommandResult = backfill.start()
+    assert backfill_start_result.success
+    backfill_scale_result: CommandResult = backfill.scale(units=10)
+    assert backfill_scale_result.success
```
Comment (chelma) [inline TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/backfill_tests.py:66]: Thank you!

I'm curious how you tested this, though.  Can you provide some details on that?

Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/backfill_tests.py:66]: I ran the Jenkins pipeline on this branch

Comment (chelma) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/backfill_tests.py:66]: To clarify - @lewijacn confirmed that the test failed at the assertions when the underlying RFS operation failed.

**Review by chelma — APPROVED** [review]

**Pull request merged**